pkg/cli/cvd.go

```diff
@@ -278,18 +278,103 @@ func (c *cvdCreator) createCVDFromAndroidCI() ([]*hoapi.CVD, error) {
 }
 
 func (c *cvdCreator) createWithCanonicalConfig() ([]*hoapi.CVD, error) {
+	hostSrv := c.client.HostService(c.opts.Host)
+	envConfig := make(map[string]any)
+	deepCopy(c.opts.EnvConfig, &envConfig)
```

Comment:
Why make a copy? Is `c.opts.EnvConfig` used for anything else after this?

Reply:
I don't have a strong opinion here, I'm just biased on keeping a reference to the original config through out the whole execution.